edenai_apis/apis/deepgram/deepgram_api.py

```
@@ -564,7 +572,9 @@ def audio__tts(
         payload = {"text": text}
 
         try:
-            response = requests.post(base_url, headers=headers, json=payload, params=params)
+            response = requests.post(
```

Ruff (0.14.14)

Probable use of `requests` call without timeout

(S113)